build/vsts-ci.yml

```diff
@@ -86,26 +86,26 @@ phases:
     continueOnError: false
     condition: and(succeeded(), in(variables._SignType, 'real', 'test'))
 
-  - script: ./build.cmd -$(BuildConfig)
+  - script: ./build.cmd -buildNative -$(BuildConfig) -buildArch=x86
     displayName: Build
   
   - task: MSBuild@1
-    displayName: Sign Windows Binaries
+    displayName: Sign Windows_x86 Binaries
     inputs:
       solution: build/sign.proj
       msbuildArguments: /p:SignType=$(_SignType)
       msbuildVersion: 15.0
     continueOnError: false
 
   - task: PublishBuildArtifacts@1
-    displayName: Publish Windows package assets
+    displayName: Publish Windows_x86 package assets
     inputs:
       pathToPublish: $(Build.SourcesDirectory)/bin/obj/packages
       artifactName: PackageAssets
       artifactType: container
 
 ################################################################################
-- phase: Windows_x86
```

Comment:
should this one be `Windows_x64`?

Reply:
Yes that's a good point, I will change the name so that it's clearer.